BUILDING.md

```diff
@@ -527,6 +528,29 @@ $  ./configure --debug --enable-asan && make -j4
 $ make test-only
 ```
 
+#### Speeding up frequent rebuilds when developing
+
+If you plan to frequently rebuild Node.js, especially if using several branches,
+installing `ccache` can help to greatly reduce build times. Setup with:
+```console
+$ sudo apt install ccache   # for Debian/Ubuntu, included in most Linux distros
```

Comment:
Probably a good idea to include how to install ccache on max and windows as well in a comment?

```
brew install ccache
choco install ccache 
```

And maybe link to https://ccache.dev/download.html ?

Reply:
OSX is flawless, but support for Visual Studio is far from perfect, I don't know if we should recommend to people to use it?